src/wifi.rs

```diff
@@ -799,6 +799,49 @@ impl<'d> WifiDriver<'d> {
         })
     }
 
+    /// Get information of AP which the ESP32 station is associated with.
+    /// Usefull to get the current signal strength of the AP, in a simple way.
+    pub fn get_ap_info(&mut self) -> Result<AccessPointInfo, EspError> {
+        let mut ap_info_raw: wifi_ap_record_t = wifi_ap_record_t::default();
+        // If Sta not connected throws EspError(12303)
+        esp!(unsafe { esp_wifi_sta_get_ap_info(&mut ap_info_raw) })?;
+        let ap_info: AccessPointInfo = Newtype(&ap_info_raw).into();
+
+        info!("AP Info: {:?}", ap_info);
```

Comment:
change this to `debug!`

Reply:
fixed in: f2bc5dd9b10a6710997a82cbe6bcf8d1297e4897